package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Javascript generated site",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. [ [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) ]

Modify this section to something like this code below : 
``` 
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

Reply:
Oops! I missed that. Thanks for pointing out!